src/pages/publication/contributors_tab/AddContributorModal.tsx

```diff
@@ -46,7 +46,8 @@ const AddContributorModal: FC<AddContributorModalProps> = ({
       }
       onClose={handleCloseModal}
       openModal={open}
-      maxWidth="lg">
+      fullWidth
+      maxWidth="md">
```

Comment:
nødvendig med `fullWidth`-prop her, når du har `maxWidth`?

Reply:
stemmer forresten for `maxWidth="sm"`, synes den blir så dratt ut når det er `"md"` (960px)

Reply:
når vi ikke lenger har hardkodet bredde for modal content med
```
min-width: 40rem;
max-width: 40rem;
```
må vi angi maxWidth, så bruker den så mye bredde den trenger (opp mot max). Siden vi bruker samme modal for å søke etter ny forfatter med og uten søketreff setter jeg den til en fixed bredde for å slippe at den hopper opp og ned i bredde. Satte fingeren i været og kom fram til at "md" var greit nok for begge tilfellene.

Vi kan evt legge inn en minWidth prop også da, hvis du vil de skal ha ulik bredde

Reply:
Jeg ser det med at det er irriterende at søkeresultat hopper i bredde, men synes det er så unødvendig stort bare. Jeg føler jeg bruker mye lengre tid på å få med meg innholdet når det er spredt utover skjermen enn når det står rett foran, hvis du skjønner?

Forsåvidt en interessant artikkel her: https://uxplanet.org/best-practices-for-modals-overlays-dialog-windows-c00c66cddd8c